doc/setup.md

```diff
@@ -147,9 +134,102 @@ Which will generate logs entries like this (assuming LOGBack pattern does not co
 ```
 
 
+#### Configure LOGBack
+
+When using the SLF4J logger, update the Cassandra ```logback.xml``` file to define path and rolling policy
+for generated audit logs.
+Add an appender and logger in your ```logback.xml``` configuration.
+The logger name of the audit records is ```ECAUDIT```.
+
+Tuning tips:
+* The asynchronous appender can _improve or demote_ performance depending on your setup.
+* Compression on rolling files may impact performance significantly.
+* If you are logging large volumes of data, make sure your storage can keep up.
+
```

Comment:
TED: "Compression on rotated" - what? I guess it should be "files"
Or should the term "rolling file(s)" be used? (that is used for the Chronicle case, and is also the from the config below - rollingPolicy).
the phrase "rotation" is also used a few lines below...

Reply:
Rephrase to use term "rolling"